tests/foreman/cli/test_user.py

```diff
@@ -429,26 +431,27 @@ def test_positive_personal_access_token_user_with_role(self, default_sat):
         :expectedresults:
             1. When used with the correct role and end point, corresponding
                output should be displayed.
-            2. When an incorrect role and end point is used, missing
+            2. When an incorrect end point is used, missing
                permission should be displayed.
 
         :CaseLevel: System
 
         :CaseImportance: High
         """
         user = make_user()
-        User.add_role({'login': user['login'], 'role': 'View hosts'})
+        User.add_role({'login': user['login'], 'role': 'Viewer'})
         token_name = gen_alphanumeric()
         result = User.access_token(
             action="create", options={'name': token_name, 'user-id': user['id']}
         )
-        token_value = result[0]['message'].split('\n')[-1]
+        token_value = result[0]['message'].split(':')[-1]
         command_output = default_sat.execute(
-            f'curl -k -u {user["login"]}:{token_value} {default_sat.url}/api/v2/hosts'
+            f'curl -k -u {user["login"]}:{token_value} {default_sat.url}/api/v2/users'
```

Comment:
For future reference, consider using `requests` library to make these HTTP requests, instead of making them via curl through SSH from the satellite host.  I don't think the coverage needs to include making the call _from satellite_.